cypress/integration/focus-trap-demo.spec.js

```diff
@@ -381,22 +423,98 @@ describe('focus-trap', () => {
       // crucial focus-trap feature: mouse click is trapped
       verifyCrucialFocusTrapOnClicking('@hideButtonInTrap');
 
-      // deactivate trap and element outside of trap can be focused again
+      // focus can be transitioned freely when trap is deactivated
       cy.get('@hideButtonInTrap').click();
-      cy.get('@lastlyFocusedElBeforeTrapIsActivated').should('be.focused');
+      verifyFocusIsNotTrapped(cy.get('@lastlyFocusedElBeforeTrapIsActivated'));
     });
   });
 
   describe('demo: radio', () => {
-    // TODO
+    it('radio group in active trap can have its value changed', () => {
+      cy.get('#demo-radio').as('testRoot');
+
+      // activate trap and 1st element in focus should be focused
+      cy.get('@testRoot')
+        .findByRole('button', { name: 'activate trap' })
+        .as('lastlyFocusedElBeforeTrapIsActivated')
+        .click();
+      cy.get('@testRoot').findByRole('radio', { name: 'b' }).as('radioB');
+      verifyCrucialFocusTrapOnClicking('@radioB');
+
+      /*
+       radio group value can be changed
+      */
+
+      // Cypress limitation to radio group value change: keyboard arrow keys can't change value
+      // so forcibly change the value by `check` command
+      //    cy.findByRole('group', { name: 'Radio group in trap' }).type('{downArrow}');
+      //    cy.get('@radioB').type('{downArrow}');
+      cy.get('@testRoot')
+        .findByRole('radio', { name: 'c' })
+        .as('radioC')
+        .check()
+        .should('be.checked');
+
+      // 'Tab' in trap should only focus the checked radio item without changing radio group value
+      cy.get('@radioC').focus();
+      cy.tab();
+      cy.get('@testRoot')
+        .findByRole('button', { name: 'deactivate trap' })
+        .as('deactivateElInTrap')
+        .should('be.focused');
+      cy.tab();
+      cy.get('@radioC').should('be.focused');
+
+      // focus can be transitioned freely when trap is deactivated
+      cy.get('@deactivateElInTrap').click();
+      verifyFocusIsNotTrapped(cy.get('@lastlyFocusedElBeforeTrapIsActivated'));
+    });
   });
 
   describe('demo: iframe', () => {
-    // TODO
+    it('focus can be moved freely in the iframe inside of trap', () => {
+      cy.get('#demo-iframe').as('testRoot');
+
+      // activate trap and element trap should be focused
+      cy.get('@testRoot')
+        .findByRole('button', { name: 'activate trap' })
+        .as('lastlyFocusedElBeforeTrapIsActivated')
+        .click();
+      cy.get('@testRoot')
+        .findByRole('button', { name: 'nothing' })
+        .as('firstElInTrap')
+        .should('be.focused');
+      verifyCrucialFocusTrapOnClicking('@firstElInTrap');
+
+      /*
+       focus can move into/out of iframe in trap
+       Due to cypress limitation(see below), `tab` is not tested
+      */
+
+      // Cypress limitation: `cy.tab()` skips focus on <iframe>
+      // cy.tab();
```

Comment:
Cypress just does not move focus into iframe. Instead in this test trying my best to ensure focus is trapped in the `iframe` by clicking on button.